usdevs/usc-website
# Updated 20th USC MC

## Description of Change
Updated 20th USC MC members, committees and images.

## Link To Issue 


## How to test this change
Run yarn start and see if the photos are updated from the 19th USC on the About Us page.

## Checklists
### Checklist for developer of feature
- [ ] I have tested my code against the Travis CI, and it has passed.

### Checklist for reviewers
- [ ] I have followed the given steps to test the PR, and it passes.
